### DISCUSSION TITLE
Possible frame-drop on MacOS ?

### DISCUSSION THREAD

**Discussion by jay19240** [gpuweb/gpuweb#4610] [Q&A]
Hello, i'm facing to a performance issue on this [demo](https://jay19240.github.io/).
The demo is pretty simple, it is intentional to help finding the problem, some events are disabled, multi-rendering texture target disabled etc...

If you move (arrow keys) left or right you can see little jump in the rendering, it is not perfectly smooth, however the profiler don't give me information and stay with a stable fps to 120.

This problem only occured on MacOSX, Windows work fine.
Can you tell me if metrics is normal ?
What can be the origin of this issue ?

Thanks so much for your regards, source code [here](https://github.com/jay19240/WARME-Y2K/tree/issue_investigation/src/samples/fps)

Chrome version: 124.0.6367.93
Macbook pro M1

**Comment by kainino0x** [comment]
As this is a Chrome issue, please file it on https://crbug.com instead, and include your `chrome://gpu` report, captured right after reproducing the bug.
